### ISSUE TITLE
should Database.EnsureCreated() be used if Database.Migrate() is being used.????

### ISSUE THREAD

**Issue by cer** [eventuate-examples/eventuate-tram-core-dotnet-examples-customers-and-orders#8]
https://github.com/eventuate-examples/eventuate-tram-core-dotnet-examples-customers-and-orders/blob/a5b7065bab554c30017b8532302edf9db5397b87/CustomerService/DBContext/CustomerContext.cs#L14

From the docs for Database.Migrate():

```
Note that this API is mutually exclusive with DbContext.Database.EnsureCreated(). EnsureCreated does not use migrations
///         to create the database and therefore the database that is created cannot be later updated using migrations.
```

**Comment by charansingh83sh** [comment]
Yes EnsureCreated does not work with migrations. I was using Database.EnsureCreated() before implementing Database Migrations and Forgot to remove this after implementing Migrations. Sorry for that. I have removed it now.
https://github.com/eventuate-examples/eventuate-tram-core-dotnet-examples-customers-and-orders/pull/10/commits/4fd5628fe8579248b1715a6ac7a62495972d308e
